test/jest/victory-bar/victory-bar.test.js

```diff
@@ -3,14 +3,42 @@
 import React from "react";
 import { render, fireEvent, screen } from "@testing-library/react";
 import { range } from "lodash";
+import { VictoryChart } from "victory-chart";
 import { VictoryBar, Bar } from "victory-bar";
 import { isBar, getBarHeight } from "../../svg-test-helper";
+import "@testing-library/jest-dom";
```

Comment:
I don't see this added to `devDependencies` yet ... maybe it's a "transient" dependency already?  In my PR, I also configured it globally using `setupFilesBeforeEnv`, maybe you can cherry-pick that config?

Reply:
(you don't really have to cherry-pick, it's fine to just wait for my PR to merge)

Reply:
I'll just fix this once your PR gets merged.